### PR TITLE
Patch: cookie widget

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- treat edge case when widget area overscrolls

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/widgets/layout/src/components/layout-widget.tsx [hunk 1]
```diff
@@ -246,7 +246,7 @@ const Layout: React.FC<unknown> = () => {
               />
             </Stack>
 
-            <Stack customStyle="fixed bottom-2 mx-2 lg:(static my-4)">
+            <Stack customStyle="fixed bottom-2 mx-2 lg:(w-[21rem] mb-2)">
```
Comment (didd) [inline ui/widgets/layout/src/components/layout-widget.tsx]: Not sure if giving this a fixed width is good idea may be max width on the entire widgets container works better. And in that case the position need to change to absolute and give the widget container position relative. This needs some thought for a better approach. Curious to see what others think here.

**Review by themonster2015 — CHANGES_REQUESTED** [review]
Screenshot 2024-05-10 at 04 29 42

It looks like with this approach, when scrolling down a feed, the latest topics and profile widgets disappear leaving a big empty space

File: extensions/widgets/layout/src/components/layout-widget.tsx [hunk 2]
```diff
@@ -234,7 +234,7 @@ const Layout: React.FC<unknown> = () => {
             </Stack>
           </Stack>
 
-          <Stack align="center" justify="between" customStyle="sticky top-0 h-screen">
+          <Stack align="center" justify="between" customStyle="sticky top-0 h-screen relative">
```
Comment (didd) [inline extensions/widgets/layout/src/components/layout-widget.tsx]: relative and sticky are mixed here, I understand the relative part is a left over from your local test. And that should fix Yen's observation.

**Review by themonster2015 — APPROVED** [review]

File: ui/widgets/layout/src/components/layout-widget.tsx (hunk 1 above)
Comment (themonster2015) [reply ui/widgets/layout/src/components/layout-widget.tsx]: Just saw the comment. I agree that giving a fixed width may not be a good idea
